### ISSUE TITLE
Return Queries that Include Base URL

### ISSUE THREAD

**Issue by mdrxy** [shlinkio/shlink-web-client#1275]
### Summary

The search bar at `/list-short-urls/` only returns the URL if you search for the slug after removing the base URL.

E.g. to get `https://hostname.tld/EF9Ze` you need to search for `EF9Ze`

My request is to return results that include the base URL, e.g. using the previous example `https://hostname.tld`

### Use case

Allows users to query short URLs quicker

**Comment by acelaya** [comment]
How is adding more text "quicker"? 😅

**Comment by mdrxy** [comment]
Referring to #1274, since we can't click the URL at the stats page, nor is the custom title displayed (#1273), one of the remaining "fast" ways to arrive at the link record page (to edit tags and other metadata) is to copy its "full" short-url to get to it. It's not that users "add" more text, rather, they'd need to in fact remove text themselves to query the short-URL's edit page.

**Comment by acelaya** [comment]
Ok, so by looking at all the issues you just opened, I'm starting to see an obvious [XY problem](https://en.wikipedia.org/wiki/XY_problem) here 😄 

Basically you had a problem (problem X), not knowing what short URL produced a specific visit, tried to work around it with the existing capabilities in shlink-web-client, almost made it, but found some limitations (problem/s Y).

Then, instead of opening an issue to see how to solve X, you opened a few smaller issues to see how to solve Y.

Without the context of "X", those "Y" issues you opened didn't seem super obvious to me at first. I usually ask for "Use case" so that people can explain how an average Shlink user would benefit from the requested feature, but nobody every provides the right info there, so I need to give it a thought.

So long story short. What you need is a way to know [the short URL that produced a visit](https://github.com/shlinkio/shlink/issues/2180). I think that's a pretty useful feature, so let's focus on that.

As per modifying the search as you requested here, it's very tricky, because the search box is "open"/"free", and it searches in multiple places (title, long URL, tags, etc). If someone actually wants the domain/base URL to be taken into consideration, this would break it for them.
